pages/common/sops.md

```diff
@@ -11,6 +11,10 @@
 
 `sops -d {{path/to/myfile.enc.json}}`
 
+- Update the declared keys in .sops.yaml
```

Comment:
```suggestion
- Update the declared keys in .sops.yaml:
```

Reply:
Hi, welcome to TlDr, as the bot said you have to add a : at the end of each command description